### PR TITLE
Edits on documentation page

### PR DESCRIPTION
**Pull request by terrytangyuan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -939,37 +939,44 @@
...
-                  <p>ElasticDL 通过实现一个 Kubernetes-native 的框架，调用 TensorFlow 2.0， 来实现弹性深度学习。</p>
+                  <h3>Elastic Scheduling and Fault-Tolerance on Kubernetes</h3>
+                  <p>Through Kubernetes-native design, ElasticDL enables fault-tolerance and works with the priority-based preemption of Kubernetes to achieve elastic scheduling for deep learning tasks.</p>
                 </section>
 
                 <section class="feature">
-                  <h3>支持 TensorFlow 2.0</h3>
-                  <p>ElasticDL 通过调用 TensorFlow 2.0 API 可以很直接地获取 gradients。</p>
+                  <h3>Support for TensorFlow 2.0</h3>
+                  <p>ElasticDL is implemented on top of TensorFlow 2.0 API that focuses on simplicity and ease of use with support for eager execution and intuitive high-level model building APIs.</p>
```
Comment (wangkuiyi) [inline index.html:948]: Let us do not use the passive voice here.

> A distributed deep learning framework needs to know local gradients before the model update. Eager Execution allows ElasticDL to do it without hacking into the graph execution process.

File: index.html [hunk 2]
```diff
@@ -939,37 +939,44 @@
...
-                  <h3>基于 Kubernetes-native 实现的弹性调度和容错</h3>
-                  <p>ElasticDL 通过实现一个 Kubernetes-native 的框架，调用 TensorFlow 2.0， 来实现弹性深度学习。</p>
+                  <h3>Elastic Scheduling and Fault-Tolerance on Kubernetes</h3>
+                  <p>Through Kubernetes-native design, ElasticDL enables fault-tolerance and works with the priority-based preemption of Kubernetes to achieve elastic scheduling for deep learning tasks.</p>
                 </section>
 
                 <section class="feature">
-                  <h3>支持 TensorFlow 2.0</h3>
-                  <p>ElasticDL 通过调用 TensorFlow 2.0 API 可以很直接地获取 gradients。</p>
+                  <h3>Support for TensorFlow 2.0</h3>
```
Comment (wangkuiyi) [inline index.html:947]: It is not "support for TensorFlow". How about 

> TensorFlow 2.0 Eager Execution

File: index.html [hunk 3]
```diff
@@ -939,37 +939,44 @@
                 </div>
 
                 <section class="feature">
-                  <h3>基于 Kubernetes-native 实现的弹性调度和容错</h3>
-                  <p>ElasticDL 通过实现一个 Kubernetes-native 的框架，调用 TensorFlow 2.0， 来实现弹性深度学习。</p>
+                  <h3>Elastic Scheduling and Fault-Tolerance on Kubernetes</h3>
```
Comment (wangkuiyi) [inline index.html:942]: How about removing "on Kubernetes"?

File: index.html [hunk 4]
```diff
@@ -939,37 +939,44 @@
...
-                  <h3>支持 TensorFlow 2.0</h3>
-                  <p>ElasticDL 通过调用 TensorFlow 2.0 API 可以很直接地获取 gradients。</p>
+                  <h3>Support for TensorFlow 2.0</h3>
+                  <p>ElasticDL is implemented on top of TensorFlow 2.0 API that focuses on simplicity and ease of use with support for eager execution and intuitive high-level model building APIs.</p>
                 </section>
 
                 <section class="feature">
-                  <h3>极简的 API 和使用方式</h3>
-                  <p>ElasticDL 与 Keras API 相比，不但原生分布式训练能力，还支持Keras不具备的精简、容错和弹性调度。</p>
+                  <h3>Simple but Flexible High-level API</h3>
```
Comment (wangkuiyi) [inline index.html:952]: How about "Minimalism interface"?

File: index.html [hunk 5]
```diff
@@ -939,37 +939,44 @@
...
+                  <p>ElasticDL enables model building through Keras API and provides simple CLI to submit distributed deep learning tasks on Kubernetes with fault-tolerance and elastic scheduling.</p>
```
Comment (wangkuiyi) [inline index.html:953]: How about shorten this description into

> Given a model defined with Keras API, train the model with a command line.

File: index.html [hunk 6]
```diff
@@ -939,37 +939,44 @@
...
+--evaluation_data_dir=/data/mnist/test \
+--num_epochs=2 \
+--master_resource_request="cpu=400m,memory=1024Mi" \
+--worker_resource_request="cpu=400m,memory=2048Mi" \
+--minibatch_size=64 \
+--records_per_task=100 \
+--num_workers=2 \
+--grads_to_wait=2 \
+--job_name=test-mnist-train \
+--output=elasticdl-saved-model
```
Comment (wangkuiyi) [inline index.html:967]: How about

```
elasticdl train mnist_functional_api.custom_model /data/mnist/train /data/mnist/test --output=the_model_dir
```

**Review by wangkuiyi — APPROVED** [review]
Approve with a few suggestions
